<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="icon" href="./images/icon.png">
<link rel="stylesheet" href="main.css" type="text/css" />
<link rel="stylesheet" href="font-awesome/css/font-awesome.min.css">
<!--- <title></title> --->
<title>Guanao Yan</title>
<!-- MathJax -->
<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-MML-AM_CHTML' async>
</script>
<script type="text/x-mathjax-config">
MathJax.Hub.Config({
	  TeX: { equationNumbers: { autoNumber: "AMS" } }
});
</script>
<!-- End MathJax -->
<div id="layout-content"> <!-- nomenu  -->
<div id="text-img-container"><div id="img-container">
<img src="./images/Guanao.JPG" alt="profile pic" width="210px" height="210px" /></div>
<div id="text-container"><h1>Guan'ao Yan <br /></h1>
<h3>Assistant Professor<br /></h3>
<p><a href="https://cmse.msu.edu/" target=&ldquo;blank&rdquo;>Department of Computational Mathematics, Science &amp; Engineering (CMSE)</a><br />
<a href="https://msu.edu/" target=&ldquo;blank&rdquo;>Michigan State University</a><br /></p>
<p><a href="dist/CV.pdf" target=&ldquo;blank&rdquo;>CV</a><br />
<a href="https://scholar.google.com/citations?user=2luz2twAAAAJ&amp;hl=en" target=&ldquo;blank&rdquo;>Google Scholar</a><br />
Email: <a href="mailto:yanguan3@msu.edu" target=&ldquo;blank&rdquo;>yanguan3@msu.edu</a><br />
Office: Engineering Building, Room 2507B</p>
</div></div>
<p>I am a tenure-track assistant professor in the Department of Computational Mathematics, Science &amp; Engineering (CMSE) at Michigan State University. I received my Ph.D. in Statistics at the University of California, Los Angeles (UCLA), where I was advised by <a href="https://jsb-lab.org/people/jingyi-jessica-li/" target=&ldquo;blank&rdquo;>Dr. Jingyi Jessica Li</a>. I hold an M.S. in Statistics from Zhejiang University, where I worked with Dr. Yi Zhang, and dual bachelor's degrees in Mathematics and Economics from Shandong University.</p>
<p>My research focuses on developing statistical methods to address challenges in real-world data analysis. My areas of interest include:</p>
<ol>
<li><p><b>Statistical Bioinformatics</b></p>
<ol>
<li><p>Statistical methods for analyzing high-dimensional single-cell and spatial omics data</p>
</li>
<li><p>Using synthetic data to enhance the statistical rigor in single-cell and spatial omics data analysis </p>
</li></ol>
</li>
<li><p><b>General Statistical Methodologies</b>: High-dimensional model inference and variable selection</p>
</li>
<li><p><b>Statistics in Education</b>: Statistical method for promoting education equity (reported by <a href="https://www.forbes.com/sites/dereknewton/2021/04/19/a-powerful-new-solution-to-collusion-cheating-and-why-we-need-it/?sh=7974b0b11d18" target=&ldquo;blank&rdquo;>Forbes</a>)</p>
</li>
</ol>
<h2>&#11088; Prospective Students</h2>
<p>I am now seeking self-motivated <b>Ph.D. students</b> and <b>M.S./undergraduate researchers</b> with interests in Statistics and/or Computational Biology. If you are excited about developing statistical or computational approaches and applying them to scientific questions, I encourage you to get in touch. </p>
<p>For perspective Ph.D. students, please (i) fill out <a href="https://forms.gle/XqoAHxu8KBUtALkD8" target=&ldquo;blank&rdquo;>this form</a> <b>and</b> (ii) email me your CV with the subject line: “Ph.D. Application: FirstName LastName”.</p>
<p>For perspective M.S./undergraduate researchers, please email me your CV, unofficial transcripts, and—if applicable—a brief statement of research interests, with the subject line: “MS/Undergraduate Researcher Application: FirstName LastName”.</p>
<h2>Publications</h2>
<p>author*: equal distribution</p>
<p>Z. Li, Z. M. Patel, D. Song, S. N. Yasa, R. Cannoodt, <b>G. Yan</b>, J. J. Li and L. Pinello (2025). <a href="https://doi.org/10.1186/s13059-025-03731-2" target=&ldquo;blank&rdquo;>Systematic benchmarking of computational methods to identify spatially variable genes.</a> <i>Genome Biology</i>, 26, 285.</p>
<p>E. Huang, T. Fu, L. Zhang, <b>G. Yan</b>, R. Yamamoto, S. Terrazas, T.L. Nguyen, C. Gonzalez-Figueroa, A. Khanbabaei, J.H. Bahn, R. Varada, K. Amoah, J. Hervoso, M.T. Paulsen, B. Magnuson, M. Ljungman, J.J. Li, and X. Xiao (2025). <a href="https://www.nature.com/articles/s41588-025-02326-8" target=&ldquo;blank&rdquo;>Genetic variants affecting RNA stability influence complex traits and disease risk.</a> <i><b>Nature Genetics</b></i>, 1-11.</p>
<p><b>G. Yan</b>, S. Hua and J.J. Li (2025). <a href="https://doi.org/10.1038/s41467-025-56080-w" target=&ldquo;blank&rdquo;>Categorization of 34 computational methods to detect spatially variable genes from spatially resolved transcriptomics data.</a> <i><b>Nature Communications</b></i>, 16, 1141.</p>
<p><b>G. Yan</b>, J.J. Li and M. Biggin (2024). <a href="https://arxiv.org/abs/2407.07420" target=&ldquo;blank&rdquo;>Question-Score Identity Detection (Q-SID): A statistical algorithm to detect collusion groups with error quantification from exam question scores.</a> <i><b>arXiv</b></i>. (Under review at <i>Journal of the American Statistical Association</i>) <a href="http://shiny2.stat.ucla.edu/Q-SID/" target=&ldquo;blank&rdquo;>[Website]</a> <a href="https://www.forbes.com/sites/dereknewton/2021/04/19/a-powerful-new-solution-to-collusion-cheating-and-why-we-need-it/?sh=1b171e9f1d18" target=&ldquo;blank&rdquo;>[Forbes article]</a> <a href="https://podcastthescore.com/episode-005-dr-karen-symms-gallagher-and-dr-mark-biggin/" target=&ldquo;blank&rdquo;>[Podcast]</a> </p>
<p>J. Zhao, F. Lao, <b>G. Yan</b> and Y. Zhang (2024). <a href="https://doi.org/10.1016/j.jik.2024.100514" target=&ldquo;blank&rdquo;>How data heterogeneity affects innovating knowledge and information in gene identification: A statistical learning perspective.</a>  <i><b>Journal of Innovation & Knowledge</b></i>, 9-3.</p>
<p><b>G. Yan</b>, D. Song and  J.J. Li (2023). <a href="https://doi.org/10.1038/s41467-023-43162-w" target=&ldquo;blank&rdquo;>scReadSim: a single-cell RNA-seq and ATAC-seq read simulator.</a> <i><b>Nature Communications</b></i>, 14(1), 7428. <a href="https://github.com/Dominic7227/scReadSim.git" target=&ldquo;blank&rdquo;>[Software]</a> <a href="https://screadsim.readthedocs.io/en/latest" target=&ldquo;blank&rdquo;>[Website]</a></p>
<p>D. Song, Q. Wang, <b>G. Yan</b>, T. Liu and J.J. Li (2023). <a href="https://www.nature.com/articles/s41587-023-01772-1" target=&ldquo;blank&rdquo;>scDesign3 generates realistic in silico data for multimodal single-cell and spatial omics.</a> <i><b>Nature Biotechnology</b></i>, 1-6. <a href="https://github.com/SONGDONGYUAN1994/scDesign3" target=&ldquo;blank&rdquo;>[Software]</a></p>
<p>S. Tang, H. Wang, <b>G. Yan</b>, L. Zhang (2022). <a href="https://link.springer.com/article/10.1007/s00184-022-00866-1" target=&ldquo;blank&rdquo;>Empirical likelihood based tests for detecting the presence of significant predictors in marginal quantile regression.</a> <i><b>Metrika</b></i>, 1-31.</p>
<p>S. Chen*, <b>G. Yan</b>*, W. Zhang, J. Li, R. Jiang and Z. Lin (2021). <a href="https://www.nature.com/articles/s41467-021-22495-4" target=&ldquo;blank&rdquo;>RA3 is a reference-guided approach for epigenetic characterization of single cells.</a> <i><b>Nature Communications</b></i>, 12(1), 1-13. <a href="https://github.com/cuhklinlab/RA3.git" target=&ldquo;blank&rdquo;>[Software]</a></p>
<p>J. Zhao, <b>G. Yan</b> and Y. Zhang (2021). <a href="https://link.springer.com/article/10.1007/s00362-021-01227-2" target=&ldquo;blank&rdquo;>Robust estimation and shrinkage in ultrahigh dimensional expectile regression with heavy tails and variance heterogeneity.</a> <i><b>Statistical Papers</b></i>, 1-28.</p>
<p>J. Zhao*, <b>G. Yan</b>* and Y. Zhang (2019). <a href="https://arxiv.org/abs/1908.06431" target=&ldquo;blank&rdquo;>Semiparametric expectile regression for high-dimensional heavy-tailed and heterogeneous data.</a> <i><b>arXiv</b></i>. (Accepted at <i>Applied Mathematics-A Journal of Chinese Universities</i>)</p>
<h2>Patents </h2>
<p>Mark Douglas Biggin, Jingyi Li, <b>Guan'ao Yan</b>. Systems and methods for detecting collusion in student testing using graded scores or answers for individual questions (Serial No. 17/450,984; <b>US Patent 11,915,615 B2</b>, filed Feburary 17, 2024)</p>
<h2>Presentations</h2>
<h3>Oral Presentations</h3>
<ul>
<li><p>STATGEN 2025, Minneapolis, USA (05/2025)</p>
</li>
<li><p>Jonsson Comprehensive Cancer Center Gene Regulation Seminar, Los Angeles, USA (05/2025)</p>
</li>
<li><p>Department of Computational Mathematics, Science &amp; Engineering, Michigan State University, USA (03/2025)</p>
</li>
<li><p>NHGRI Genome Tech Dev Coordinating Center Working Group, Jackson Laboratory, USA (01/2025)</p>
</li>
<li><p>Department of Mathematical Sciences, New Jersey Institute of Technology, USA (12/2024)</p>
</li>
<li><p>Department of Statistics and Data Sciences, University of California, Los Angeles, USA (11/2024)</p>
</li>
<li><p>Joint Statistical Meetings, Portland, USA (08/2024)</p>
</li>
<li><p>Institute for Computational and Experimental Research in Mathematics, Providence, USA (12/2023)</p>
</li>
<li><p>Jonsson Comprehensive Cancer Center Gene Regulation Seminar, Los Angeles, USA (11/2023)</p>
</li>
<li><p>Institute for Quantitative and Computational Biosciences Research Seminar, Los Angeles, USA (12/2022)</p>
</li>
<li><p>NSF-Simons Center for Multiscale Cell Fate 5th Annual Symposium, Irvine, USA (10/2022)</p>
</li>
<li><p>The 7th International Conference on Statistics and Probability, IMS-China, Dalian, China (07/2019)</p>
</li>
</ul>
<h3>Poster Presentations</h3>
<ul>
<li><p>Cold Spring Harbor Laboratory Genome Informatics Conference, New York, USA (12/2023)</p>
</li>
<li><p>RECOMB/ISCB Conference on Regulatory &amp; Systems Genomics, Los Angeles, USA (11/2023)</p>
</li>
<li><p>Chan Zuckerberg Initiative Single-Cell Biology 2023 Annual Meeting, Carlsbad, USA (11/2023)</p>
</li>
<li><p>ISMB/ECCB, Lyon, France (07/2023)</p>
</li>
<li><p>Los Angeles Bioscience Ecosystem Summit, Los Angeles, USA (05/2023)</p>
</li>
<li><p>Jonsson Comprehensive Cancer Center Retreat Poster Session, Los Angeles, USA (05/2023)</p>
</li>
<li><p>Institute for Quantitative and Computational Biosciences Poster Session, Los Angeles, USA (09/2022)</p>
</li>
</ul>
<h2>Teaching</h2>
<h3>Instructor</h3>
<ul>
<li><p>CMSE 201 <i>Computational Modeling and Data Analysis I</i> (Fall 2025) <a href="https://msu-cmse-courses.github.io/cmse201-f25-jb/intro.html" target=&ldquo;blank&rdquo;>[Website]</a></p>
</li>
</ul>
<h3>Teaching assistant</h3>
<ul>
<li><p>UCLA STATS 205 <i>Hierarchical Linear Models</i> (Spring 2024)</p>
</li>
<li><p>UCLA STATS 203 <i>Large Sample Theory</i> (Winter 2024)</p>
</li>
<li><p>ElevatePro <i>Statistical Science with Applications to Epidemiology</i> (Summer 2021)</p>
</li>
<li><p>ZJU MATH 1001 <i>Advanced Mathematics</i> (Fall 2018 & 2019)</p>
</li>
</ul>
<h3>Workshop instructor</h3>
<ul>
<li><p>Presenter, Jonsson Comprehensive Cancer Center Workshop, UCLA (Dec 2024)<br />
<i>&ldquo;Categorization of 34 Computational Methods to Detect Spatially Variable Genes from Spatially Resolved Transcriptomics Data&rdquo;</i> </p>
</li>
<li><p>Coordinator &amp; Presenter, QCBio Workshop, UCLA (May 2022)<br />
<i>“Statistical Methods for Enhancing the Rigor in Single-cell RNA-seq Data Analysis”</i> </p>
</li>
</ul>
<h3>Guest lecturer</h3>
<ul>
<li><p>UCLA STATS 205 <i>Hierarchical Linear Models</i> (Spring 2024)</p>
</li>
<li><p>UCLA BIOINFO 229 <i>Current Topics in Bioinformatics</i> (Winter 2024)</p>
</li>
</ul>
<h3>Undergraduate student mentor</h3>
<ul>
<li><p>Weijian Wang, Undergraduate Student at Zhejiang University (2022 - 2025), Currently PhD student at UCLA </p>
</li>
<li><p>Zhiyin Liu, Undergraduate Student at Hong Kong University of Science and Technology (2022 - 2025), Currently PhD student at UCLA </p>
</li>
<li><p>Shuo Hua, Undergraduate Student at Tsinghua University (June to Dec 2022), Next MS in Computer Science at Stanford </p>
</li>
</ul>
<p><br />
<br />
&ldquo;<i>If we all worked on the assumption that what is accepted as true is really true, there would be little hope of advance.</i>&rdquo; <p style="text-align: right">- Orville Wright</p></p>
</div> <!-- <div id="layout-content"> -->
<div id="footer-container">
<div id="footer">
<div id="footer-text">
Last edited  on November 6<sup>th</sup> 2025  04:51PM (Time Zone: EST). </br>
</div> <!-- <div id="footer-text"> -->
</div> <!-- <div id="footer"> -->
</div> <!-- <div id="footer-container"> -->
<div> <!-- nomenulastbit to counteract  <div id="layout">  -->
</div> <!--- <div id="layout"> --->
</div> <!--- <div id="main-container"> --->
<script>
function openNav() {
    if (window.innerWidth <= 1200) {
        document.getElementById("layout-menu").style.width = "280px";
        document.getElementById("layout-content-container").style.marginLeft = "280.8px";
        document.getElementById("layout-content-container").style.position = "fixed";
    }
}
function closeNav() {
    if (window.innerWidth <= 1200) {
        document.getElementById("layout-menu").style.width = "0";
        document.getElementById("layout-content-container").style.position = "static";
        document.getElementById("layout-content-container").style.marginLeft = "0px";
        setInterval(
            function(){ location.reload() },
            500
        );
    }
}
</script>
</body>
</html>
